### PR TITLE
Fix codenvy maven

### PR DESCRIPTION
**Pull request by wentzt**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -37,8 +46,8 @@
       <version>2018.2.2</version>
     </dependency>
   </dependencies>
+
```
Comment (jforevers) [inline pom.xml:49]: nit. extra newline

File: pom.xml [hunk 2]
```diff
@@ -51,4 +60,5 @@
       <url>http://www.kauailabs.com/maven2</url>
     </repository>
   </repositories>
+
 </project>
```
Comment (jforevers) [inline pom.xml:63]: nit. add newline

File: pom.xml [hunk 3]
```diff
@@ -51,4 +60,5 @@
       <url>http://www.kauailabs.com/maven2</url>
     </repository>
   </repositories>
+
```
Comment (jforevers) [inline pom.xml:62]: nit. extra newline

**Review by clebio — APPROVED** [review]
:shipit:

File: pom.xml (hunk 3 above)
Comment (wentzt) [reply pom.xml:62]: I tried to fix it, but it didn't take my change, idk.

**Pull request merged**